src/graphics/texture.js

```diff
@@ -970,6 +968,23 @@ class Texture {
 
         return buff;
     }
+
+    // get the texture's encoding type
+    get encoding() {
+        if (this.type === TEXTURETYPE_RGBM) {
+            return 'rgbm';
+        }
+
+        if (this.type === TEXTURETYPE_RGBE) {
+            return 'rgbe';
+        }
+
+        if (this.format === PIXELFORMAT_RGBA16F || this.format === PIXELFORMAT_RGBA32F) {
+            return 'linear';
+        }
+
+        return 'srgb';
```

Comment:
this is not correct in general case I think .. linear vs sRGB should be a separate flag / getter.

Reply:
I know what you mean, but encoding is only ever applied when dealing with textures used for lighting. For example normal maps are never 'decoded'.

Reply:
maybe it could be renamed to `lightingEncoding()`?

Reply:
I think I'm more partial to `encoding`, TBH.